src/role_upgrader.js

```diff
@@ -14,7 +14,7 @@ roles.upgrader.stayInRoom = true;
 roles.upgrader.buildRoad = false;
 roles.upgrader.killPrevious = true;
 
-roles.upgrader.boostActions = ['upgradeController'];
+//roles.upgrader.boostActions = ['upgradeController'];
```

Comment:
It doesn't work for upgraders? These were my first creeps to enable, everything else later, you want to get GCL